### PR TITLE
Home dropdown

### PR DESCRIPTION
**Pull request by walknt**
Added referencing and fixed the conflict

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -293,6 +294,80 @@ input:checked + .disabledSlider:before {
     font-family: "Fira Mono";
 }
 
+/* Style for Autocomplete dropdown from Geoapify Autocomplete tutorial */
+/* URL: https://apidocs.geoapify.com/samples/autocomplete/autocomplete-tutorial/ */
+
+.autocomplete-container {
```
Comment (ando527) [inline css/styles.css:300]: change to upperCamelCase please

File: css/styles.css [hunk 2]
```diff
@@ -293,6 +294,80 @@ input:checked + .disabledSlider:before {
...
+  
+  .autocomplete-container input {
+    width: calc(100% - 43px);
+    padding: 10px;
+    border: 1px solid rgba(0, 0, 0, 0.2);
+    font-size: 16px;
+    outline: none;
+  }
+  
+  .autocomplete-items {
```
Comment (ando527) [inline css/styles.css:313]: camelCase please

File: scripts/profile.js [hunk 3]
```diff
@@ -10,9 +10,8 @@ var distanceCheck;
 var nameText;
 var profilePic;
 
-
 $( document ).ready(function() {
-    addressBox = document.querySelector("#addressBox");
+    addressBox = document.querySelector("#autocomplete-container");
```
Comment (ando527) [inline scripts/profile.js:14]: camelCase

**Review by ando527 — CHANGES_REQUESTED** [review]
Can we please use camelCase for the variable names to stay consistent across the site

File: scripts/profile.js (hunk 3 above)
Comment (walknt) [reply scripts/profile.js]: camelCase added 👍

**Review by ando527 — APPROVED** [review]

**Pull request merged**